src/ImageRunModal.jsx

```diff
@@ -647,7 +645,7 @@ export class ImageRunModal extends React.Component {
         }
 
         const localImage = this.state.image || (selectedImage && this.props.localImages.some(img => img.Id === selectedImage.Id));
-        const registries = this.props.registries && this.props.registries.search ? this.props.registries.search : utils.fallbackRegistries;
+        const podmanRegistries = registries && registries.search ? registries.search : utils.fallbackRegistries;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1249-20230710-125104-d428b4fe-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)